surveymonkey/surveymonkey.py

```diff
@@ -96,6 +159,77 @@ def context(self):
             "introductory_text": self.introductory_text,
             "text_link": self.text_link,
             "survey_link": link,
+            "completed_survey": self.completed_survey,
+            "record_completion": self.record_completion,
         }
 
         return context
+
+    def _validate_survey_name(self, validation, data, api_surveymonkey):
+        """
+        This method searches survey list by survey title and sets a message if the survey is not unique
+        """
+        kwargs = {"title": data.survey_name}
+        response = api_surveymonkey.get_surveys(**kwargs)
+        data_response = response.get("data", [])
+
+        filtered_data = [survey for survey in data_response if survey["title"] == data.survey_name]
+        count = len(filtered_data)
+
+        if count == 0:
+            validation.add(ValidationMessage(ValidationMessage.ERROR, u"Invalid survey name, the survey doesn't exist"))
+            return
+        elif count > 1:
+            validation.add(ValidationMessage(
+                ValidationMessage.ERROR, u"Invalid survey name, there are two or more surveys with the same name"
+            ))
+            return
+
+        collectors = api_surveymonkey.get_collectors(filtered_data[0].get("id"), **{"include": "url,type"})
+        data_collectors = collectors.get("data")
+
+        if len(data_collectors) == 0:
```

Comment:
I think `if not len(data_collectors)` would be better

Reply:
check